cmd/boulder-ca/main.go

```diff
@@ -176,6 +177,17 @@ func main() {
 		cmd.FailOnError(err, "Failed to create Publisher client")
 	}
 
+	if c.CA.GRPC != nil {
+		s, l, err := bgrpc.NewServer(c.CA.GRPC, scope)
+		cmd.FailOnError(err, "Unable to setup VA gRPC server")
```

Comment:
I think this is a copy pasta error. Do you mean "Unable to setup CA gRPC server"?